### PR TITLE
REM-703: Wrap rpc error response

### PR DESCRIPTION
**Pull request by DeV1doR**
Implements issue REM-703.

### CODE REVIEW

File: remme/shared/exceptions.py [hunk 1]
```diff
@@ -12,27 +12,46 @@
...
+class ValidatorNotReadyException(RemmeRpcError):
 
-class CountInvalid(ClientException):
-    pass
+    MESSAGE = 'Validator is not ready yet'
+    ERROR_CODE = -32002
+
+
+class ResourceHeaderInvalid(RemmeRpcError):
+
```
[the remark this answers is not in the document]
Comment (DeV1doR) [reply remme/shared/exceptions.py]: Not a pep8 error, better readability

**Comment by DeV1doR** [comment]
@eugene-babichenko do not agree that all methods must be enabled, because it could be harmful (as example export_node_key and set). If u use a node and what to use some module, better is explicitly used. All default modules already enabled.

**Comment by eugene-babichenko** [comment]
Disagree on this point. I think we should discuss some standards :)

**Comment by eugene-babichenko** [comment]
We can remove this method from the production setup. Anyway disabling all methods can lead to a lot of talk on additional configuration steps from those who will run nodes only for personal use.

**Comment by DeV1doR** [comment]
So in this case, wildcard still need)

**Comment by eugene-babichenko** [comment]
No wildcards please :)

Let’s do the following

* Enable _everything_ in the repository;
* Add documentation with a special notice on security;
* See how it works.

**Comment by eugene-babichenko** [comment]
Still having this (disabled `pkc` in settings).

image

File: remme/shared/exceptions.py [hunk 2]
```diff
@@ -12,27 +12,46 @@
...
+class ClientException(RemmeRpcError):
 
-class ValidatorNotReadyException(Exception):
-    pass
+    ERROR_CODE = -32000
 
 
-class ResourceHeaderInvalid(ClientException):
-    pass
+class KeyNotFound(RemmeRpcError):
```
[the remark this answers is not in the document]
Comment (eugene-babichenko) [reply remme/shared/exceptions.py]: Moved to a separate task.

**Comment by DeV1doR** [comment]
> Still having this (disabled `pkc` in settings). Everything else is OK.
> 
> image

It in personal module

**Pull request merged**